lib/iex/lib/iex/helpers.ex

```diff
@@ -1260,10 +1262,11 @@ defmodule IEx.Helpers do
   ## Examples
 
       iex> ref("0.1.2.3")
-      #Reference<0.1.2.3>
+      ~Reference<0.1.2.3>
 
   """
   @doc since: "1.6.0"
+  # TODO: deprecate in favour of ~Reference sigil
```

Comment:
We should likely put the version of the deprecation. I think we can soft-deprecate in 1.11, meaning we'd make this `@doc false` and then change this to 
```suggestion
  # TODO: hard-deprecate in favour of ~Reference sigil in Elixir v1.13
```

Thoughts @josevalim?